modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -1209,2824 +1208,1603 @@ class SampleCodeTest {
+        
+        try TLSIdentity.deleteIdentity(withLabel: "doco-sync-server")
+        
+        // end::p2p-tlsid-delete-id-from-keychain[]
+        
+    }
+    
+    func dontTestReplicatorConfigSelfCert() throws {
+        let target = DatabaseEndpoint(database: otherDB)
+        var config = ReplicatorConfiguration(database: database, target: target)
+        // tag::old-p2p-act-rep-config-self-cert[]
+        // acceptOnlySelfSignedServerCertificate = true -- accept Slf-Signed Certs
+        //        FIXME: !!
+        //          `disableTLS` is for listener and it is 'false' by default.
+        //          `acceptOnlySelfSignedServerCertificate` is for replicator
+        //        config.disableTLS = false
+        config.acceptOnlySelfSignedServerCertificate = true
+        
+        // end::old-p2p-act-rep-config-self-cert[]
+    }
+    
+    // tag::p2p-act-rep-config-cacert-pinned-func[]
+    func myCaCertPinned() {
+        let targetURL = URL(string: "wss://10.1.1.12:8092/otherDB")!
+        let targetEndpoint = URLEndpoint(url: targetURL)
+        var config = ReplicatorConfiguration(database: database, target: targetEndpoint)
+        // tag::p2p-act-rep-config-cacert-pinned[]
+        
+        // Get bundled resource and read into localcert
+        guard
+            let pathToCert = Bundle.main.path(forResource: "listener-pinned-cert", ofType: "cer"),
+            let localCertificate:NSData = NSData(contentsOfFile: pathToCert)
+        else { /* process error */ return }
+        
+        // Create certificate
+        // using its DER representation as a CFData
+        guard
+            let pinnedCert = SecCertificateCreateWithData(nil, localCertificate)
+        else { /* process error */  return }
+        
+        // Add `pinnedCert` and `acceptOnlySelfSignedServerCertificate=false` to `ReplicatorConfiguration`
+        config.acceptOnlySelfSignedServerCertificate = false
+        config.pinnedServerCertificate = pinnedCert
+        // end::p2p-act-rep-config-cacert-pinned[]
+        // end::p2p-act-rep-config-cacert-pinned-func[]
+        
+    }
+    
+    func dontTestOldListenerConfigClientRootCA() throws {
+        // tag::old-listener-config-client-root-ca[]
+        // Configure the client authenticator to validate using ROOT CA <.>
+        // end::old-listener-config-client-root-ca[]
+    }
+    
+    // end::p2p-tlsid-manage-func[]
+    
+    // tag::replication-start-func[]
+    enum PeerConnectionStatus: UInt8 {
+        case stopped = 0;
+        case offline
+        case connecting
+        case idle
+        case busy
+    }
+    
+    func dontTestReplicationStart(_ peer: String,
+                                  peerDBName: String,
+                                  user: String?,
+                                  pass: String?,
+                                  handler: @escaping (PeerConnectionStatus, Error?) -> Void) throws {
+        guard let userDb = self.database else {
```

Comment:
Can we assume that self.database is always be there for code snippet and just use self.database?